README.old.md

```diff
@@ -0,0 +1,34 @@
+# LAB - Class 24 <br>
+<br><br>
+## Project Name: Routing and Component Composition<br>
+<br><br>
+Author: Sian Culligan<br>
+<br><br>
+Links and Resources<br>
+submission PR<br>
+[PR](https://github.com/SianCulligan/401_lab24/pull/1)<br>
+<br><br>
+Deployment Links <br>
+[Coming soon]()
+<br><br>
+
+Setup<br>
+.env requirements (where applicable)<br>
+SKIP_PREFLIGHT_CHECK=true<br>
+
+<br><br>
+UML<br>
+[UML](.assets/Lab24UML.png)
```

Comment:
`![UML](./assets/Lab24UML.png)` to get the photo to show. If you want to leave as a link just take off the bang at the begining.